### DISCUSSION TITLE
Edit page hide fields if some field not true

### DISCUSSION THREAD

**Discussion by jouniikaheimo** [orchidsoftware/platform#1788] [Q&A]
Hi

it possible hide fields example: i have member edit form and there is checkbox company if that uncheck that hide company name and company address etc.. ? 

this possible to implement

**Comment by tabuna** [comment]
You can do this with a listener. It is described in the [documentation](https://orchid.software/en/docs/listener/), and you can also see it in action in the [video](https://youtu.be/mXFPooGSqs4)
